app/views/application/notes/index.njk

```diff
@@ -55,10 +55,10 @@
           {% endfor %}
         </div>
       {% else %}
-        <p>There are no notes yet. We’ll guide you through creating your first note.</p>
+        <p>Notes are only seen by you and colleagues. We'll guide you through note creation.</p>
 
         {{govukButton({
-          text: "Add note",
+          text: "Continue",
```

Comment:
I don’t think continue is very clear here. It doesn’t make sense without first reading the leading paragraph which isn’t guaranteed.

Reply:
Hmm, isn't the first step clicking on the Notes tab though? :)

As it stands, I still think we're asking someone to 'Add note' before they understand what that does...

Reply:
It could be that they intend to add a note. But just as likely is that they want to view a note (or edit it).

>  As it stands, I still think we're asking someone to 'Add note' before they understand what that does...

Since the last time we tested this, we’ve added an interstitial and we’ve added this little lede in above the button. I think that’s a good improvement for now and we can monitor in the next rounds of research.